doc/modules/kernel_approximation.rst

```diff
@@ -59,13 +59,13 @@ a linear algorithm, for example a linear SVM::
     >>> y = [0, 0, 1, 1]
     >>> rbf_feature = RBFSampler(gamma=1, random_state=1)
     >>> X_features = rbf_feature.fit_transform(X)
-    >>> clf = SGDClassifier()   # doctest: +NORMALIZE_WHITESPACE
+    >>> clf = SGDClassifier(tol=0.)   # doctest: +NORMALIZE_WHITESPACE
```

Comment:
Was this really necessary here?

Also, using "tol=0" in an example seems like a bad habit to give to a user. The tol should be set to a reasonable value, not zero. And if people are using SGD, I would expect that they don't care about an excellent convergence, as this is not in general the best algorithm for that.